libcst/_nodes/expression.py

```diff
@@ -1670,6 +1698,14 @@ class Parameters(CSTNode):
     #: Optional parameter that captures unspecified kwargs.
     star_kwarg: Optional[Param] = None
 
+    #: Positional-only parameters, with or without defaults. Positional-only
+    #: parameters with defaults must all be after those without defaults.
+    posonly_params: Sequence[Param] = ()
```

Comment:
The Parameters class becomes fairly complex after this change. It worth provide some examples with mixed usage of those different type of parameter.